modules/sdk-lib-mpc/src/util.ts

```diff
@@ -137,3 +138,22 @@ export async function randomPositiveCoPrimeLessThan(x: bigint): Promise<bigint>
 export async function randomBigInt(bitlength: number): Promise<bigint> {
   return bigIntFromBufferBE(Buffer.from(await randBits(bitlength, true)));
 }
+
+/**
+ * @param seed - used to construct derivation path deterministically
+ * @param isMaster - if set, path starts with prefix `m/`
+ * @return path `(m/)/999999/a/b` where `a` and `b` are 7-byte pseudorandom numbers based on seed
+ */
+export function getDerivationPath(seed: string, isMaster = true): string {
+  function sha256(input: crypto.BinaryLike): Buffer {
+    return crypto.createHash('sha256').update(input).digest();
+  }
```

Comment:
Can we move this function outside? There is no reason to re-create this function every time we call `getDerivationPath`

Reply:
It can remain a a private/un-exported function, but define outside of `getDerivationPath`